generator/test/good_test_data/embedded_annotations.expected

```diff
@@ -0,0 +1,104 @@
+  }) {
+    RealmObjectBase.set(this, 'single child', child);
+    RealmObjectBase.set<RealmList<Child1>>(
+        this, 'CHILDREN', RealmList<Child1>(children));
+  }
+
+  Parent._();
+
+  @override
+  Child1? get child =>
+      RealmObjectBase.get<Child1>(this, 'single child') as Child1?;
+  @override
+  set child(covariant Child1? value) =>
+      RealmObjectBase.set(this, 'single child', value);
+
+  @override
+  RealmList<Child1> get children =>
+      RealmObjectBase.get<Child1>(this, 'CHILDREN') as RealmList<Child1>;
+  @override
+  set children(covariant RealmList<Child1> value) =>
+      throw RealmUnsupportedSetError();
+
+  @override
+  Stream<RealmObjectChanges<Parent>> get changes =>
+      RealmObjectBase.getChanges<Parent>(this);
+
+  @override
+  Parent freeze() => RealmObjectBase.freezeObject<Parent>(this);
+
+  static SchemaObject get schema => _schema ??= _initSchema();
+  static SchemaObject? _schema;
+  static SchemaObject _initSchema() {
+    RealmObjectBase.registerFactory(Parent._);
+    return const SchemaObject(ObjectType.topLevel, Parent, 'Parent', [
+      SchemaProperty('single child', RealmPropertyType.object,
+          mapTo: 'single child', optional: true, linkTarget: 'MySuperChild'),
+      SchemaProperty('CHILDREN', RealmPropertyType.object,
+          mapTo: 'CHILDREN',
+          linkTarget: 'MySuperChild',
+          collectionType: RealmCollectionType.list),
+    ]);
+  }
+}
+
+class Child1 extends _Child1 with RealmEntity, RealmObjectBase, EmbeddedObject {
+  Child1(
+    String value,
+    String indexedString, {
+    Parent? linkToParent,
+  }) {
+    RealmObjectBase.set(this, '_value', value);
+    RealmObjectBase.set(this, '_parent', linkToParent);
+    RealmObjectBase.set(this, 'indexedString', indexedString);
+  }
+
+  Child1._();
+
+  @override
+  String get value => RealmObjectBase.get<String>(this, '_value') as String;
+  @override
+  set value(String value) => RealmObjectBase.set(this, '_value', value);
+
+  @override
+  Parent? get linkToParent =>
+      RealmObjectBase.get<Parent>(this, '_parent') as Parent?;
+  @override
+  set linkToParent(covariant Parent? value) =>
+      RealmObjectBase.set(this, '_parent', value);
+
+  @override
+  String get indexedString =>
+      RealmObjectBase.get<String>(this, 'indexedString') as String;
+  @override
+  set indexedString(String value) =>
+      RealmObjectBase.set(this, 'indexedString', value);
+
+  @override
+  Stream<RealmObjectChanges<Child1>> get changes =>
+      RealmObjectBase.getChanges<Child1>(this);
+
+  @override
+  Child1 freeze() => RealmObjectBase.freezeObject<Child1>(this);
+
+  static SchemaObject get schema => _schema ??= _initSchema();
+  static SchemaObject? _schema;
+  static SchemaObject _initSchema() {
+    RealmObjectBase.registerFactory(Child1._);
+    return const SchemaObject(ObjectType.embedded, Child1, 'MySuperChild', [
+      SchemaProperty('_value', RealmPropertyType.string, mapTo: '_value'),
```

Comment:
Could you check `SchemaProperty` api doc to the class and the property mapTo. It is written that it is for RealmObject, maybe should be RealmObject and EmbeddedObject.

Reply:
```dart
/// Describes a [RealmObject]'s property with its name, type and other attributes in the [RealmSchema]
///{@category Configuration}
class SchemaProperty {
```
could be:
```dart
/// Describes a property of [RealmObject] or [EmbeddedObject] with its name, type and other attributes in the [RealmSchema]
///{@category Configuration}
class SchemaProperty {
```